apps/core/views/viewsets/comment.py

```diff
@@ -55,7 +57,18 @@ class CommentViewSet(mixins.CreateModelMixin,
     }
 
     def create(self, request, *args, **kwargs):
-        return super().create(request, *args, **kwargs)
+        if self.request.data['parent_article'] is None:
+            parent_article = Comment.objects.get(pk=self.request.data['parent_comment']).parent_article
```

Comment:
이렇게 하면, 사용자가 parent_comment에 이상한 id를 줬을 때 get()이 None 나와서 에러가 날 것 같아요.
아래 전체적으로 리팩토링해서 적어보았습니다.

Reply:
`get()` method는 실패 시 `DoesNotExist` 에러가 발생합니다.
`get_object_or_404()` 함수를 이용해서 아래와 같이 고쳤습니다.
```suggestion
            comment_queryset = Comment.objects.all()
            parent_comment = get_object_or_404(comment_queryset, pk=self.request.data['parent_comment'])
            self.check_object_permissions(request, parent_comment)
            parent_article = parent_comment.parent_article
```